README.md

```diff
@@ -25,18 +25,16 @@
 ## Introduction
 
 The SDS SDK contains artifacts and resources for all parties that intent to use, extend or integrate with the BAMM
-Aspect Meta Model, e.g. Solution Developers, Domain Experts or OEMs.
+Aspect Meta Model, e.g., Solution Developers, Domain Experts or OEMs.
 
-At its core are components, which help working with the BAMM Aspect Meta Model (BAMM). The SDK comprises components to
-load and validate models such as the one depicted below and generate artifacts such as DTOs, static classes,
-documentation and so on.
-![BAMM Aspect Meta Model (BAMM) Elements](documentation/developer-guide/modules/getting-started-guide/images/solution-with-movement-aspect.svg)
+At its core are components, which help to work with the BAMM Aspect Meta Model (BAMM). The SDK comprises components to
+load and validate models generate artifacts such as static classes, documentation, DTO (data transfer object) and so on.
```

Comment:
"DTOs (data transfer objects)"? As "static classes" is in plural, DTOs should maybe also be plural